### DISCUSSION TITLE
Font does not load

### DISCUSSION THREAD

**Discussion by eerison** [LunarVim/LunarVim#3542] [Q&A] [answered]
Hello, I'm trying to create a docker image to lunarvim but atm I'm blocked in fonts

I download the font, and after run `fc-cache` it finds the font, but it is not working :(

```
fc-cache -f -v
/usr/share/fonts: caching, new cache contents: 0 fonts, 1 dirs
/usr/share/fonts/truetype: caching, new cache contents: 0 fonts, 1 dirs
/usr/share/fonts/truetype/dejavu: caching, new cache contents: 6 fonts, 0 dirs
/usr/local/share/fonts: caching, new cache contents: 0 fonts, 0 dirs
/home/dev/.local/share/fonts: caching, new cache contents: 1 fonts, 0 dirs
/home/dev/.fonts: skipping, no such directory
/usr/share/fonts/truetype: skipping, looped directory detected
/usr/share/fonts/truetype/dejavu: skipping, looped directory detected
/var/cache/fontconfig: not cleaning unwritable cache directory
/home/dev/.cache/fontconfig: cleaning cache directory
/home/dev/.fontconfig: not cleaning non-existent cache directory
fc-cache: succeeded
```
https://github.com/shield-wall/lunarvim-docker/blob/master/Dockerfile#L28-L31

But it looks like this into the docker container

![Screenshot from 2022-11-28 15-06-18](https://user-images.githubusercontent.com/6358755/204297539-a02b0548-c3ad-455f-8f86-6ce9de0f2761.png)

**Comment by LostInTheLogs** [comment]
Does it only happen in docker?

**Reply by eerison** [reply]
I just tested using docker, I didn't test in my machine :/

**Comment by eerison** [comment]
just to let the discussion in one place: https://github.com/LunarVim/LunarVim/issues/3486#issuecomment-1329199140

even if I'm into the container I must install into the host?

**Reply by LostInTheLogs** [reply]
the container doesn't render the fonts, the terminal does and it runs on the host

**Reply by eerison** [reply]
ok i got it! I will test :)

**Comment by eerison** [comment]
@LostNeophyte 

some icons that wasn't showing, now it's showing ❤️ 
![Screenshot from 2022-11-28 21-15-02](https://user-images.githubusercontent.com/6358755/204372484-b8a25168-8967-4c7f-9744-299fbf625463.png)

But the icons in the main page is not working, am I missing to install something else?
I installed this font: https://www.lunarvim.org/docs/configuration/nerd-fonts#curl-download
![Screenshot from 2022-11-28 21-08-41](https://user-images.githubusercontent.com/6358755/204372379-1548da12-2863-48ac-aaf0-75aace901b71.png)

**Comment by LostInTheLogs** [comment, marked as answer]
Have you set this font as your current one in the terminal?

28 Nov 2022, 21:18 by ***@***.***:

**Reply by eerison** [reply]
Ops ... an user issue 😅 
Thank you so much, it worked as expected ❤️
